.github/workflows/rust.yml

```diff
@@ -81,6 +76,14 @@ jobs:
           command:                 clippy
           toolchain:               ${{ matrix.toolchain }}
           args:                    --all-targets -- -D warnings
+
+## Test stage
+      - name:                      Testing rust-${{ matrix.toolchain }}
```

Comment:
The reason for tests being before clippy was to get quicker feedback about failing tests instead of getting a bunch of annoying `clippy` complaints :) But I guess we will still get that from the `stable` job.

Reply:
I think it depends on which pipeline complains first. I suspect Clippy may be fast to fail than the tests (since we need to wait for _everything_ to compile first). If we find this annoying in our workflow we can change the order back to the original one